news/2017-09-21-beatmap-spotlights-july-august-2017.md

```diff
@@ -0,0 +1,259 @@
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/sayonara-heaven.jpg)](https://osu.ppy.sh/s/516995)
+
+With such clean and consistent mapping, **(jieusieu)[https://osu.ppy.sh/u/759439]**, **[Beomsan](https://osu.ppy.sh/u/3626063)** and **[Enon](https://osu.ppy.sh/u/2043401)** managed to create a wonderful mapset worthy of this month's Spotlight.
+
+Each difficulty captures the song wonderfully while also putting attention to the breaks in the music, and the map itself flows brilliantly, topped off with some excellent and beatifully fitting hitsounding.
+
+Not only that, this mapset also contains a fantastic storyboard by someone who decided to remain anonymous. Definitely worth giving this mapset a try which maintains a gradual difficulty spread for players of all types.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/petals.jpg)](https://osu.ppy.sh/s/564354)
+
+**[toybot](https://osu.ppy.sh/u/2848604)** graces us once more with a charming rendition of Camellia's and Nanahira's latest artcore collaboration, Petals. The structure and patterns presented in the map are clean and polished, with a characteristic toybot stacking and blanketing style.
+
+The rhythms are given enough thought to ensure they both represent the song, as well as allowing ample variation to keep player's on the edge of their seat. Supported by a dazzling storyboard by **[Kanari](https://osu.ppy.sh/u/891115)**, this map beautifully captures the wonderful voice of Nanahira, while maintaining a secondary focus on the instrumental track of the song! Definitely deserves a place in this month's Spotlights!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/heroine-syndrome.jpg)](https://osu.ppy.sh/s/603694)
+
+While it may not be innovative, **[Delis](https://osu.ppy.sh/u/1603923)** has presented us with yet another well made map this month filled, with his nostalgia-focused design choices. What makes Delis's maps stand out from the masses of modern maps is their old-style object composition and slider design, which resemble maps from an age most current osu! players have never gotten a chance to see.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/soumatou-labyrinth.jpg)](https://osu.ppy.sh/s/625493)
+
+The winner of the osu!taiko spotlights this month is no one else but **[komasy](https://osu.ppy.sh/u/1980256)**! Soumatou Labyrinth provides an extremely solid and consistent spread from Kantan to Inner Oni which will satisfy players of any skill level. Special shoutout to the Inner Oni that makes use of very intuitive 1/6 snappings while keeping an overall high difficulty with the usage of complex rhythms, suiting the song perfectly. A well deserved first place!
```

Comment:
probably Spotlights needs capitalization on the S

1/6 snappings → 1/6 patterns or patterning